src/App.tsx

```diff
@@ -5,7 +5,8 @@ import { CssBaseline } from '@material-ui/core'
 import { Header, Navigation } from 'components'
 import { Login } from 'pages'
 
-import { LOGIN_PAGE, HOME } from 'constants/routes'
+import { LOGIN_PAGE, HOME, PROFILE } from 'constants/routes'
+import Profile from 'pages/Profile'
```

Comment:
You can import `Profile` straight from `pages/index.ts`, just export it there. You can see Login as an example, when you export `Profile` in `pages/index.ts` you can import it here like this:
```ts
import { Login, Profile } from 'pages'
```

That's a good practice to group imports because files can get messy quite quickly when importing with full routes. There are some files that use 20 components so you know 😄